### PR TITLE
Improve the KO message on how to proceed for CLA

### PR DESCRIPTION
**Pull request by elicoidal**

### CODE REVIEW

**Comment by elicoidal** [comment]
@gurneyalex

File: clabot.ini [hunk 1]
```diff
@@ -30,10 +30,15 @@ odoo_icla_categ =
...
-cla_ko_message = Hey @{pull_user}, thank you for your Pull Request.
+cla_ko_message = Hey @{pull_user}, thank you for your Pull Request and contribution to the OCA.
 
     It looks like some users haven't signed our **C**ontributor **L**icense **A**greement, yet.
-    You can read and sign our full Contributor License Agreement here: http://odoo-community.org/page/website.cla
+    
+    1. You can get our full Contributor License Agreement (CLA) here: http://odoo-community.org/page/website.cla
+    2. Your company (with Enterprise CLA) or every users listed below (with Individual CLA) should complete and sign it
+    3. Please scan the document(s) and send them back to cla@odoo-community.org,
+    4. Do not forget to include the Github login(s).
```
Comment (dreispt) [inline clabot.ini]: Should #4 rather be a sentence in #3 ?

**Comment by elicoidal** [comment]
@dreispt @gurneyalex

**Review by dreispt — APPROVED** [review]

**Comment by elicoidal** [comment]
@gurneyalex  can we move forward?

**Comment by elicoidal** [comment]
up?

**Comment by elicoidal** [comment]
I suggest to close this PR @gurneyalex

**Comment by elicoidal** [comment]
:tada: :smile:

**Pull request merged**